travel-planner/travel-planner-ui/src/Components/Marker/Marker.jsx

```diff
@@ -121,7 +89,7 @@ export default function Marker(props) {
 		return () => {
 			marker.setMap(null);
 		};
-	}, [props]);
+	}, []);
```

Comment:
why props are removed from the dependencies?

Reply:
The marker's props will never change while the site is running so there is no reason for that. It just needs to render the marker and its onClick function once. I had props here before since for some reason it never ran without it. This has been fixed.